ui/history.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Fast-Food-Fast</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="css/style.css" />
+    <script src="js/main.js"></script>
+</head>
+<body>
+    <div id = "menu-bar">
+        <a href="index.html">Home</a>
+        <a href="order.html">Order</a>
+        <a href="history.html">Order History</a>
+        <span id="right-menu"><a href="signup.html">Sign Up</a></span>
+        <span id="right-menu"><a href="login.html">Login</a></span>
```

Comment:
@66hakeem you are using id: **right-menu** more than once however it should be used only once in a doc, if it is to be used more than once, use classes instead